sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -114,6 +115,7 @@ protected SchemaTransform from(KafkaReadSchemaTransformConfiguration configurati
     consumerConfigs.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, autoOffsetReset);
 
     String format = configuration.getFormat();
+    boolean failOnError = configuration.getErrorHandling() == null;
```

Comment:
Use `!ErrorHandling.hasOutput(configuration.getErrorHandling())`. It's possible errorHandling is set, but errorHandling.output is not. (This is also more future compatible to other options that may be put here.)

Reply:
Done.